signature-aggregator/aggregator/aggregator.go

```diff
@@ -121,19 +116,10 @@ func (s *SignatureAggregator) AggregateSignaturesAppRequest(
 		return nil, errNotEnoughConnectedStake
 	}
 
-	// TODO: remove this special handling and replace with ACP-118 interface once available
-	var reqBytes []byte
-	if sourceSubnet == constants.PrimaryNetworkID {
-		req := coreEthMsg.MessageSignatureRequest{
-			MessageID: unsignedMessage.ID(),
-		}
-		reqBytes, err = coreEthMsg.RequestToBytes(coreEthCodec, req)
-	} else {
-		req := msg.MessageSignatureRequest{
-			MessageID: unsignedMessage.ID(),
-		}
-		reqBytes, err = msg.RequestToBytes(codec, req)
-	}
+	reqBytes, err := msg.RequestToBytes(
+		msg.Codec,
+		msg.MessageSignatureRequest{MessageID: unsignedMessage.ID()},
```

Comment:
This is still the legacy API. ACP-118 doesn't support requests by MessageID (although they will still be available). We should switch to using the protobuf encoding following @cam-schultz's reference implementation in avalanche-go at ava-labs/avalanchego#3218